src/components/command-bar/components/activator/index.tsx

```diff
@@ -2,14 +2,18 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: MPL-2.0
  */
-
+import { ReactElement } from 'react'
 import Badge from 'components/badge'
 import { useCommandBar } from 'components/command-bar'
 import { CmdCtrlIcon, KIcon } from 'components/command-bar/components'
 import Text from 'components/text'
-import { CommandBarActivatorProps } from './types'
 import s from './command-bar-activator.module.css'
 
+interface CommandBarActivatorProps {
+	leadingIcon: ReactElement
+	visualLabel: string
+}
```

Comment:
I am a big fan of co-locating the types here.  Low hanging fruit to reduce to total number of files you have to navigate to understand this component!

Reply:
I agree, I prefer co-locating types for simple interfaces like this.